cli/test/test_gha.py

```diff
@@ -413,7 +413,7 @@ def test_docker_image_exists(self):
         'Skipping docker tests...')
     def test_docker_rm(self):
         self.docker_client.images.pull('debian:buster-slim')
-        self.runner.docker_create('debian:buster-slim')
+        self.runner.docker_create('debian:buster-slim', None)
```

Comment:
if we make `docker_create(..., runtime_config=None)`  as mentioned above, we don't need to provide value for it